data/outfits.txt

```diff
@@ -407,18 +2730,6 @@ outfit "Nerve Gas"
 	"unplunderable" 1
 	description "This is a highly illegal weapon for hand to hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
-outfit "Knockout Gas"
```

Comment:
this did revert the knockout gas change, but thats an easy fix for me. I'll be sure to readd it myself.